packages/components/src/models/PlayingCardDeck.ts

```diff
@@ -0,0 +1,6 @@
+import type { PlayingCard } from './PlayingCard';
+
+export interface PlayingCardDeck {
```

Comment:
Could be something like

```
export interface Deck<T> {
   id: string;
   cards: T[]
}
```

to be reusable

Reply:
Makes perfect sense 😄